mods/d2k/rules/vehicles.yaml

```diff
@@ -442,7 +442,7 @@ deviator:
 		BuildPaletteOrder: 40
 		BuildDuration: 373
 		BuildDurationModifier: 40
-		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry, Aircraft\n \n  Atreides:      +Range\n  Harkonnen:  +Health\n  Ordos:          +Speed
+		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry
```

Comment:
Each combat tank is actually a different actor, i'll add them under their description. Actually that is also how i gone with in my mod.

But not now, need to wait for tomorrow, sorry.